rules/summary-recording-sprint-reviews/rule.md

```diff
@@ -1,84 +1,103 @@
 ---
 seoDescription: Enhance transparency and inclusivity by recording summaries of Sprint Reviews, providing stakeholders with clear updates and fostering community engagement.
 type: rule
-title: Do you record a summary of Sprint Reviews for the stakeholders?
+title: Do you record a summary of your Sprint Meeting?
 uri: summary-recording-sprint-reviews
 authors:
   - title: Jack Pettit
     url: https://ssw.com.au/people/jack-pettit/
+  - title: Jeoffrey Fischer
+    url: https://ssw.com.au/people/jeoffrey-fischer/
+  - title: Landon Maxwell
+    url: https://ssw.com.au/people/landon-maxwell/
   - title: Adam Cogan
     url: https://ssw.com.au/people/adam-cogan/
 related:
   - tech-check
   - record-a-quick-and-dirty-done-video
   - rules-to-better-video-recording
+  - making-a-great-done-video
 created: 2024-04-15T00:46:35.508Z
 guid: 99674806-7353-4d39-ac9a-219620d4e107
 ---
 
-Creating a comprehensive summary and recording of Sprint Reviews is a great way to communicate changes in a Product to the community and stakeholders. Especially for people who are unable to attend these meetings.
+Creating a comprehensive summary and recording of your Sprint Meeting is a great way to communicate changes in a Product to the community and stakeholders - especially for people who are unable to attend these meetings. This rule covers the following:
+
+* Why you should record a summary of the Sprint Meeting
+* How to record it (best practices)
+* How to share the recording
```

Comment:
```suggestion
Creating a comprehensive summary and recording of your Sprint Meeting is a great way to communicate changes in a product to the community and stakeholders — especially for those unable to attend.
```

Maybe something like this - I think we try to avoid saying "this rule" and listing out stuffs in the intro/blurb